tensorboard/uploader/flags_parser.py

```diff
@@ -201,3 +203,11 @@ def define_flags(parser):
     auth_revoke.set_defaults(
         **{AUTH_SUBCOMMAND_FLAG: AUTH_SUBCOMMAND_KEY_REVOKE}
     )
+
+
+def fix_flags(flags):
+    if hasattr(flags, "plugins"):
+        expanded_plugins = []
+        for plugin in flags.plugins:
+            expanded_plugins.extend(str.split(plugin, ","))
```

Comment:
It seems most confusing to support *both* comma-separated and tokenized
flag values. If we want to pick comma-separated flag values, can we drop
the `nargs="*"`?

If we’re worried about compatibility, we can always push a 2.2.1 release
with this cherry-pick before launching any features that depend on this.
Presumably we’d need to do that anyway: if we want comma-separated to be
the documented and encouraged form, then it’ll need to be published
before we use it in examples.